include/splinterdb/splinterdb.h

```diff
@@ -61,9 +61,6 @@ typedef struct splinterdb_config {
    // Shared memory support
    uint64 shmem_size;
    _Bool  use_shmem; // Default is FALSE.
-   _Bool  trace_shmem_allocs;
-   _Bool  trace_shmem_frees;
-   _Bool  trace_shmem; // Trace both allocs & frees from shared memory
```

Comment:
Withdrew these. I already had API function to set global Booleans. So, the rework was pretty simple.